src/natcap/invest/web_api.py

```diff
@@ -0,0 +1,211 @@
+"""A Flask app with HTTP endpoints used by the InVEST Workbench."""
```

Comment:
I'm wondering if the name `web_api` is misleading... What about `ui_server.py`?

Reply:
I dunno? I thought `server` was kind of inaccurate since this module defines the *app* that runs on the *server*, right? I like having `ui` in the name though. Hopefully we get some more opinions here!

Reply:
Naming things is always the hardest part! I guess I shied away from `web` because we're only running this locally and the `web` is not really involved. `http_api` seems fine to me. I don't care too deeply, so I'll be fine with whatever you choose!

Reply:
I like `http` over `web`. Maybe `ui_api` since the relevant info is that it's entrypoints for the UI, not how it's implemented (http)? But `http_api` seems fine too

Reply:
Thanks for all the input, I renamed it to `ui_server`